source/s3_copy_object.c

```diff
@@ -540,6 +540,37 @@ static int s_s3_copy_object_prepare_request(struct aws_s3_meta_request *meta_req
     return AWS_OP_ERR;
 }
 
+/* For UploadPartCopy requests, etag is sent in the request body, within XML entity quotes */
+static struct aws_string *s_get_etag_from_upload_part_copy_response(
```

Comment:
trivial: maybe rename to `s_etag_new_from_upload_part_copy_response`? Since this function creates a new `aws_string`, which will be a bit more clear about the lifetime.